easybuild/easyconfigs/c/chaco/chaco-20160817-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'chaco'
+version = '20160817'
+commit_id = 'eeae890'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://code.enthought.com/projects/chaco/'
+description = """Chaco is a Python plotting application toolkit that facilitates
+ writing plotting applications at all levels of complexity,
+ from simple scripts with hard-coded data to large plotting programs
+ with complex data interrelationships and a multitude of interactive tools.
+ While Chaco generates attractive static plots for publication and presentation,
+ it also works well for interactive data visualization and exploration."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/enthought/chaco/archive/']
+sources = ['%s.tar.gz' % commit_id]
```

Comment:
@boegel I had two options. Try to do everything with `Python3`, as the user want to use `PyBERT` simultaneously with `pyGIMLi`. As far as I got from `pyGIMLi` homepage, its requires `Python3`. So I had to use `Python3` here as well. and the latest official release is not yet `Python3` compatible. I could use version`4.6.0beta.20160817` (or whatever is the version claimed by the code). 

Other possibility to make a `Python2/2.7.11-Python-3.5.1`, and use all the stuff here with that.
(`chaco/4.5.0-foss-2016a-Python-2.7.11-Python-3.5.1` or so). The option to use system `Python` is not a good way, since `PyBERT` requires `Python 2.7`